core/tracker/inclusion.go

```diff
@@ -632,8 +646,36 @@ func (a *InclusionChecker) Run(ctx context.Context) {
 			if checkedSlot == slot {
 				continue
 			}
+			indices := []eth2p0.ValidatorIndex{}
+			a.core.mu.Lock()
+			subs := maps.Clone(a.core.submissions)
```

Comment:
We shall refactor this after the fix. Now lock+clone is definitely much slower than the new `sync.Map`.